src/openforms/forms/migrations/0013_auto_20211220_1755.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 2.2.25 on 2021-12-20 16:55
+
+from django.db import migrations
+
+
+def search_and_replace_widget(components: list) -> list:
+    for component in components:
+        if component["type"] == "select" and component["widget"] == "html5":
+            component["widget"] = "choicesjs"
+
+        if "components" in component:
+            component["components"] = search_and_replace_widget(component["components"])
```

Comment:
I think this does not cover the `fieldset` and `columns` component types! See `FormDefinition.iter_components`